Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/TracingRepository.kt

```diff
@@ -153,11 +153,10 @@ class TracingRepository @Inject constructor(
         val taskLastFinishedAt = try {
             taskController.tasks.first()
                 .filter { it.taskState.type == DownloadDiagnosisKeysTask::class }
-                .mapNotNull { it.taskState.finishedAt }
-                .sortedDescending()
-                .first()
+                .mapNotNull { if (it.taskState.finishedAt != null) it.taskState.finishedAt else it.taskState.startedAt }
```

Comment:
Maybe `it.taskState.finishedAt ?: it.taskState.startedAt`

Reply:
good point. adjusted.